sdk/ai/azure-ai-projects/README.md

```diff
@@ -1,53 +1,188 @@
 
 # Azure AI Projects client library for Python
-<!-- write necessary description of service -->
+
+Use the AI Projects client library (in preview) to:
+
+* **Enumerate connections** in your Azure AI Studio project and get connection properties.
+For example, get the inference endpoint URL and credentials associated with your Azure OpenAI connection.
+* **Get an already-authenticated Inference client** for the default Azure OpenAI or AI Services connections in your Azure AI Studio project. Supports the AzureOpenAI client from the `openai` package, or clients from the `azure-ai-inference` package.
+* **Develop agents using the Azure AI Agent Service**, leveraging an extensive ecosystem of models, tools, and capabilities from OpenAI, Microsoft, and other LLM providers. The Azure AI Agent Service enables the building of agents for a wide range of generative AI use cases. The package is currently in private preview.
+* TODO: Evaluations
```

Comment:
TODO

Reply:
Yap, I ping Ankit again to provide me the text. He also has to add examples for Evaluations.